scripts/openstack.py

```diff
@@ -0,0 +1,156 @@
+import argparse
```

Comment:
@dachary has some pretty good arguments (ha) against docopt and in favor of argparse. As much as I prefer docopt as a developer I can't argue against him using argparse here.